### ISSUE TITLE
[Accessibility] "Open quick start guide" links are not descriptive.

### ISSUE THREAD

**Issue by shahparth-lab** [Azure/LogicAppsUX#3256]
### Describe the Bug with repro steps

User Experience: 
Screen reader users will be impacted if the screen reader will not announce descriptive information of the controls present on the screen. Users will get confused and will not be able to navigate properly. 

Note: 
User credentials should NOT be included in the bug.

Repro Steps:
Open https://ms.portal.azure.com/ and login via valid credentials.
Navigate to the Left Nav bar and select 'More Services', search 'Integration Accounts' and select any already created Integration Account
Select Overview in left navigation pane. Navigate all the controls on the blade
Verify the issue.

Actual Result:
The "Open quick start guide" links are not descriptive, present on the overview screen. Screen reader is only announcing "Open quick start guide link".

Expected Result:
The "Open quick start guide" link should be descriptive, present on the overview screen. Screen reader should announce the information like "link to logic app open quick start guide link".

### What type of Logic App Is this happening in?

Consumption (Portal)

### Are you using new designer or old designer

New Designer

### Did you refer to the TSG before filing this issue? https://aka.ms/lauxtsg

Yes

### Browser

OS: Windows 11 Version 21H2 (OS build: 22000.795)
Browser: Chrome Version 103.0.5060.134 (Official Build) (64-bit)

AB#25072727

https://msazure.visualstudio.com/One/_workitems/edit/15014316
